### ISSUE TITLE
About your mobilenet+SE result on imagenet.

### ISSUE THREAD

**Issue by AmigoCDT** [kobiso/CBAM-tensorflow#4]
In SENet paper, the SE_Mobilenet top-1 accuracy on imagenet is 74.7%, but in your result, it is 70.03%, the margin between them is too large.
So, what caused this gap? optmizer, hyper-parameters?
Thanks.
